Pocket/pocket-ios
# SwiftUI Home - Analytics

## Goal
* This PR adds analytics events to SwiftUI Home

## Test Steps
* Build/run on Xcode and make sure in the console you view the same tracking events of the existing Home
